docs/topics/settings.rst

```diff
@@ -604,6 +604,26 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
+-----------------------
+
+Default: ``False``
+
+Whether or not to allow broken responses, where declared ``Content-Length``
+does not match content sent by the server. If ``False``, these responses
+trigger a ``ResponseFailed`` error. If ``True``, these responses are allowed
+and the flag ``partial`` is added.
```

Comment:
I doubt these flags were widely used, but renaming 'partial' to 'data-loss' for this case is backwards incompatible (if the answer to previous question is "yes"); I think that's fine to do this change, but it must be mentioned in release notes.